### PR TITLE
chore(deps): updates all dependencies

### PR DESCRIPTION
**Pull request by kleinfreund**
Updates all dependencies.

Removes `data-v` attributes from snapshots to avoid minor/patch releases of Kongponents to breaking our tests solely on account of all the `data-v` attributes changing.

Adds workaround plugin for Vite issue https://github.com/vitejs/vite/issues/2415.

Signed-off-by: Philipp Rudloff <philipp.rudloff@konghq.com>

### CODE REVIEW

**Comment by kleinfreund** [comment]
**Comment**: Need to tweak the removal of data-v-* attributes a bit more.

File: jest/jest-replace-attribute-snapshot-serializer.ts [hunk 1]
```diff
@@ -33,6 +33,13 @@ export function replaceAttributesSnapshotSerializer(
         }
       }
 
+      // Removes “data-v-*” attributes.
+      for (const attribute of value.attributes) {
+        if (attribute.name.startsWith('data-v-')) {
+          value.removeAttribute(attribute.name)
+        }
+      }
+
```
Comment (johncowen) [inline jest/jest-replace-attribute-snapshot-serializer.ts:42]: I understood that this is removing the data-v attributes? Wasn't sure whether i should still be seeing them in the snapshots? On a brief scan of the diff here, it looks like they've been removed from some places but not others?

**Review by johncowen — APPROVED** [review]
Looks good to me, I had a question on the data-v attributes but thats non-blocking

File: jest/jest-replace-attribute-snapshot-serializer.ts (hunk 1 above)
Comment (kleinfreund) [reply jest/jest-replace-attribute-snapshot-serializer.ts:42]: I'm struggling a bit with getting the custom serializer to go through all nodes. I got most of them removed locally but there are still a handful left. Fixing that next week.

Comment (johncowen) [reply jest/jest-replace-attribute-snapshot-serializer.ts:42]: Awesome, sounds great! Will be good once they are all gone

Comment (kleinfreund) [reply jest/jest-replace-attribute-snapshot-serializer.ts:46]: Figured it out. Much happier with removing those attributes now. This should minimize the amount of times we have to regenerate these snapshots.

**Pull request merged**